### DISCUSSION TITLE
Opacity Hotkey

### DISCUSSION THREAD

**Discussion by superkitten12** [runelite/runelite#14985] [Ideas]
This one is pretty simple, just add the option to swap between preset opacity values with a bindable hotkey or combination. I dont want to always play in 30 opacity, only when i have something behind the client, and going in to change it between 30 and 100 manually is a little tedious if i need to do it more than once every couple of hours.

**Comment by antero111** [comment]
You can do this with [Plugin Presets](https://github.com/antero111/plugin-presets). Make some presets controlling your clients opacity and bind them to your desired keybinds. There is also possibility to activate presets based on client focus so you could have 30% opacity when the client is not active and 100% when active.
